src/ActionModal/components/ActionModal.vue

```diff
@@ -830,6 +844,20 @@ export default {
     maxDecimals(value, decimals) {
       return Number(BigNumber(value).toFixed(decimals)) // TODO only use bignumber
     },
+    alternativeFeeSelector() {
```

Comment:
make this a computed

Reply:
I cannot since I overwrite `this.gasPrice` and `this.gasEstimate` in there

Reply:
which is a bad sign